lib/sgtn-client/api/translation.rb

```diff
@@ -158,5 +159,9 @@ def self.refresh_cache(component, locale)
       cache_key = SgtnClient::CacheUtil.get_cachekey(component, locale)
       @refresh_cache_operator.operate(cache_key, component, locale)
     end
+
+    String.class_eval <<-LOCALE, __FILE__, __LINE__ + 1
```

Comment:
I get the idea, putting this info in the hash item itself instead of an an additional "fallback_keys" in the cache entry looks simpler. Any other ideas though? I don't want to modify Strings class.